pkg/dbutil/common.go

```diff
@@ -189,6 +190,42 @@ func CloseDB(db *sql.DB) error {
 	return errors.Trace(db.Close())
 }
 
+// addClusteredAnnotation add the `/*T![clustered_index] NONCLUSTERED */` for primary key of create table info
+// In the older version, the create table info hasn't `/*T![clustered_index] NONCLUSTERED */`,
+// which lead the issue https://github.com/pingcap/tidb-tools/issues/678
+//
+// Before Get Create Table Info:
+// mysql> SHOW CREATE TABLE `test`.`itest`;
+//
+//	+-------+--------------------------------------------------------------------+
+//	| Table | Create Table                                                                                                                              |
+//	+-------+--------------------------------------------------------------------+
+//	| itest | CREATE TABLE `itest` (
+//		`id` int(11) DEFAULT NULL,
+//		`name` varchar(24) DEFAULT NULL,
+//		PRIMARY KEY (`id`)
+//		) ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_bin |
+//	+-------+--------------------------------------------------------------------+
+//
+// After Add the annotation:
+//
+//	+-------+--------------------------------------------------------------------+
+//	| Table | Create Table                                                                                                                              |
+//	+-------+--------------------------------------------------------------------+
+//	| itest | CREATE TABLE `itest` (
+//		`id` int(11) DEFAULT NULL,
+//		`name` varchar(24) DEFAULT NULL,
+//		PRIMARY KEY (`id`) /*T![clustered_index] CLUSTERED */
+//		) ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_bin |
+//	+-------+--------------------------------------------------------------------+
+func addClusteredAnnotationForPrimaryKey(raw string) (string, error) {
```

Comment:
I think it should check if the tidb_version() is below v5, and the table has `PKisHandle` flag.
If version < 5 {
If `PKisHandle`,
add /*T![clustered_index] CLUSTERED */,
else
add /*T![clustered_index] NONCLUSTERED */.
}